examples/platforms/da15000/radio.c

```diff
@@ -84,7 +84,11 @@ static uint32_t sleep_const_delay;
 
 static otInstance *First_Instace;
 
-#define RF_MODE_BLE             0
+uint32_t NODE_ID = 1;
```

Comment:
It was used as temporary way of calculating aIeeeEui64. Proper long term solution is under preparation.